packages/react-core/src/filters/Filter.js

```diff
@@ -1,7 +1,14 @@
 import { filterFunctions } from './FilterTypes';
+import { FiltersLogicalOperators } from '../operations/constants/FiltersLogicalOperators';
 
-function passesFilter(columns, filters, feature) {
-  return columns.every((column) => {
+const logicalOperatorMethods = {
```

Comment:
Since it's a constant (in the pure meaning of the word), the convention says that it should be LOGICAL_OPERATOR_METHODS

Reply:
Done 👍